scripts/utils/ensure-px-units.js

```diff
@@ -0,0 +1,26 @@
+/*
+Copyright © 2021 The Sage Group plc or its licensors. All Rights reserved
+ */
+const isNumber = require('lodash/isNumber')
+const isString = require('lodash/isString')
+
+/**
+ * Ensures that input has pixel units if needed
+ * @param input
+ * @returns {string|*}
+ */
+module.exports = (input) => {
+  if (isString(input) && input.includes('px')) {

```

Comment:
maybe one condition is enough ?
if (isString(input) && (input.includes('px') || input === '0' )) { ...

Reply:
It is split on purpose. I've done it for better code readability.
After all I don't know if one solution is better than the other one